clang/test/dpct/cmake_migration/case_061/expected.txt

```diff
@@ -0,0 +1,5 @@
+find_library(CUDA_LIB "" PATH ${prefix_path})
```

Comment:
2nd parameter of find_library() Specify one or more possible names for the library.   What expect to do if the name is empty?  should we remove it?

Reply:
If the lib name is empty, the var is set to **NOFOUND** value. No error is thrown.
Removing the entire statement would result in errors for the var not being defined.